crates/chia-sdk-driver/src/spend_with_conditions.rs

```diff
@@ -0,0 +1,14 @@
+use chia_sdk_types::Conditions;
+use clvm_utils::TreeHash;
+
+use crate::{DriverError, Spend, SpendContext};
+
+pub trait SpendWithConditions {
+    fn puzzle_hash(&self) -> TreeHash;
```

Comment:
Why aren't primitives just `ToTreeHash`?

Reply:
Good point, I could just add a ToTreeHash bound wherever this is used for now

Reply:
Or ToClvm<Allocator> and calculate the tree hash manually, idk